x-pack/plugins/apm/public/application/csmApp.tsx

```diff
@@ -32,6 +31,7 @@ import { createCallApmApi } from '../services/rest/createCallApmApi';
 import { px, units } from '../style/variables';
 import { createStaticIndexPattern } from '../services/rest/index_pattern';
 import { UXActionMenu } from '../components/app/RumDashboard/ActionMenu';
+import { redirectTo } from '../components/routing/render_as_redirect';
```

Comment:
Is it really necessary to call it `render_as_redirect`?
```suggestion
import { redirectTo } from '../components/routing/redirect';
```